include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_RUNTIME_HOSTMANAGERR_HOSTMANAGER_H
+#define GLOW_RUNTIME_HOSTMANAGERR_HOSTMANAGER_H
+#include "glow/Backends/Backend.h"
+#include "glow/Backends/DeviceManager.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Runtime/RuntimeTypes.h"
+
+#include <atomic>
+#include <llvm/ADT/StringRef.h>
```

Comment:
Please take a look at other places where we include LLVM. We usually place them as one chunk below the glow includes and above the system includes.